application/controllers/SurveyAdministrationController.php

```diff
@@ -1611,9 +1611,16 @@ public function actionDeactivate()
                 $this->redirect($this->createUrl("surveyAdministration/view/surveyid/{$iSurveyID}"));
             }
             if (Yii::app()->request->getPost('ok') == '') {
+                if (!empty($_SESSION['sNewSurveyTableName'])) {
+                    unset($_SESSION['sNewSurveyTableName']);
+                }
+
+                $_SESSION['sNewSurveyTableName'] = Yii::app()->db->tablePrefix . "old_survey_{$iSurveyID}_{$date}";
```

Comment:
Better to use Yii session handler. Look for App()->session in the codebase to see how it's done